psc-package/Main.hs

```diff
@@ -275,7 +296,10 @@ main = do
         , Opts.command "available"
             (Opts.info (pure listPackages)
             (Opts.progDesc "List all packages available in the package set"))
-        ]
+		, Opts.command "verify-all"
```

Comment:
Please fix the indentation here, and add a command description. Also can we change this to `verify-set` please?